### ISSUE TITLE
Costume crop produce essence with no use

### ISSUE THREAD

**Issue by Bertichram** [BlakeBr0/MysticalAgriculture#586]
I added a crop to produce Cinnebar from thermal and it worked i can craft the seeds and produce the essence.
bud i cant use the essence to craft the cinnebar gem or dust.  am i doing something wrong or is this a bug

cinnabar.json
{
    "name": "Cinnabar",
    "type": "mysticalagriculture:resource",
    "tier": "mysticalagriculture:3",
    "ingredient": {
      "tag": "forge:gems/cinnabar"
    },
    "color": "E34234",
    "textures": {
      "flower": "mysticalagriculture:block/flower_dust",
      "essence": "mysticalagriculture:item/essence_dust"
    }
  }

cucumber-tags.json
{
  "__comment": "Instructions: https://blakesmods.com/docs/cucumber/tags-config",
  "forge:dusts/niter": "thermal:niter_dust",
  "forge:dusts/cinnabar": "thermal:cinnabar_dust",
  "forge:dusts/saltpeter": "immersiveengineering:dust_saltpeter",
  "forge:rubber": "null"
}

 - Minecraft: 1.19.2
 - Modpack: ATM8

**Comment by BlakeBr0** [comment]
Yeah, you'll have to add a use for the essences yourself using either a datapack or something like crafttweaker
